pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -90,8 +96,10 @@ type LoadBalancer struct {
 // 3. Ready for work (Ready)
 func (r *ReconcileAPIScheme) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-
-	reqLogger.Info("Reconciling APIScheme", "request", request)
+	reconcileCount++
+	if reconcileCount%1000 == 0 {
+		reqLogger.Info("Reconciling APIScheme", "request", request)
+	}
```

Comment:
what is the purpose of this?

Reply:
I don't think this will carry over between reconciles.

Reply:
We don't want to log every reconcile because if we did that, it would fill the logs every minute, so this is a middle ground between zero log entries and too many.

This snippet will log a line for every 1000th reconcile, or approximately every 1000 minutes.

Reply:
@fahlmant Won't it? Up on line 36 `reconcileCount` is defined as a package variable, its value should carry?

Reply:
Ah I missed that bit.

Still, I think this is the wrong way to go. Once a minute is not that much logging. And taking out any logs for the rest of the reconcile reduces visibility into what the operator is doing to what resources.

Reply:
@fahlmant I think you're right. It's probably better to err on the side of "too much" than "not enough" because we have the option to turn the knob down later. I'll make that change here.

/hold

Reply:
/hold cancel

@fahlmant @sam-nguyen7 thoughts on c9e363e?

Reply:
Just a drive-by here...

Why is the `Reconcile` being called so frequently in the first place? Is something legitimately changing that's triggering it?

Reply:
@2uasimojo Yes, on line 221 we intentionally requeue. The purpose is to ensure that any changes to the `openshift-kube-apiserver/service/rh-api` object's `LoadBalancerSourceRanges` are reconciled with the authoritative source (the apischeme CR); see lines 142-155.